scripts/_common.sh

```
@@ -0,0 +1,680 @@
+  else
+    # level is a range name
+    level=$(echo $level | tr "A-Z" "a-z")
+    local found=0
+    for i in "${!SEVERITY_RANGES_NAMES[@]}"; do
+      if [[ "${SEVERITY_RANGES_NAMES[$i]}" == "$level" ]]; then
+        level="${SEVERITY_RANGES[$i]}"
+        found=1
+        break
+      fi
+    done
+    [[ "$found" == 0 ]] && fatal "invalid log level '$level', must be one of: ${SEVERITY_RANGES_NAMES[@]}"
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)